examples/tutorials/03_ready_to_use_model_torch.ipynb

```diff
@@ -13,15 +13,16 @@
    "metadata": {},
```

Comment:
<p>From the outside perspective, it looks like you are adding namespace to <code>before</code> pipeline. That is also not an <code>action</code>, but just a method of <code>pipeline</code>. It would be better to separate it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>No need for before and namespace at all</p>